test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,201 @@
+--stanza=seg%c archive-push %p'" --skipvalidation
+
+gpstop -ar
+
+echo "pgbackrest health check" > /dev/null
+for i in -1 0 1 2
+do 
+	PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i check
+done
+
+# The test scenario starts here
+echo "Creating initial dataset..." > /dev/null
+createdb gpdb_pitr_database
+psql -d gpdb_pitr_database -c \
+"CREATE TABLE t1 (id int, text varchar(255)) DISTRIBUTED BY (id);"
+
+psql -d gpdb_pitr_database -c \
+"INSERT INTO t1 SELECT i, 'text'||i FROM generate_series(1,30) i;"
+
+psql -d gpdb_pitr_database -c "SELECT * FROM t1 ORDER BY id;" \
+-o $PGBACKREST_TEST_DIR/$TEST_NAME/t1_rows_original.out
+
+echo "Creating full backup on master and seg0..." > /dev/null
+for i in -1 0
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i \
+    --type=full backup
+done
+
+echo "Checking the presence of first backup" > /dev/null
+function check_backup(){
+    segment_backup_dir=$PGBACKREST_TEST_DIR/$TEST_NAME/backup/seg$1
+    current_date=$(date +%Y%m%d)
+    if [[ $(find $segment_backup_dir -maxdepth 1 -type d -name \
+        "${current_date}-??????F" -not -empty ) ]];
+    then
+        echo "Found a backup directory for segment $1: $dirname" > /dev/null
+    else
+        echo "The backup directory for segment $1 was not found" > /dev/null
+        exit 1
+    fi
+}
+for i in -1 0
+do 
+   check_backup $i
+done
+
+echo "Creating additional table..." > /dev/null
+psql -d gpdb_pitr_database -c \
+"CREATE TABLE t2 (id int, text varchar(255)) DISTRIBUTED BY (id);"
+
+psql -d gpdb_pitr_database -c \
+"INSERT INTO t2 SELECT i, 'text'||i FROM generate_series(1,30) i;"
+
+psql -d gpdb_pitr_database -c "SELECT * FROM t2 ORDER BY id;" \
+-o $PGBACKREST_TEST_DIR/$TEST_NAME/t2_rows_original.out
+
+echo "Creating full backup on seg1 and seg2..." > /dev/null
+for i in 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i \
+    --type=full backup
+done
+
+echo "Checking the presence of second backup" > /dev/null
+for i in 1 2
+do 
+   check_backup $i
+done
+
+echo "Creating a distributed restore point" > /dev/null
+psql -d gpdb_pitr_database -c "create extension gp_pitr;"
+psql -d gpdb_pitr_database -c "select gp_create_restore_point('test_pitr');"
+psql -d gpdb_pitr_database -c "select gp_switch_wal();"
+
+echo "Simulating disaster..." > /dev/null
+psql -d gpdb_pitr_database -c "drop table t1;"
+psql -d gpdb_pitr_database -c "truncate table t2;"
+
+gpstop -a
+rm -rf $MASTER/* $PRIMARY1/* $PRIMARY2/* $PRIMARY3/*
+rm -rf $MIRROR1/* $MIRROR2/* $MIRROR3/* $DATADIR/standby/*
+
+echo "Restoring cluster..." > /dev/null
+for i in -1 0 1 2
+do 
+	PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest --config \
```

Comment:
replace the tab with spaces
```suggestion
  PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest --config \
```